downstream/titles/aap-operator-installation/master.adoc

```diff
@@ -9,7 +9,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Installing on Openshift Container Platform
+= Operating {PlatformNameShort}
```

Comment:
```suggestion
= Operating Ansible Automation Platform
```

Reply:
Best to avoid using attributes in titles. I usually copy / paste from docinfo.xml to ensure no typos.